README.md

```diff
@@ -1,20 +1,20 @@
-# Deploy Vue 3 example to Layer0
+# Deploy Vue 3 example to Edgio
 
-A demo deployment of Vue 3 app to Layer0.
+A demo deployment of Vue 3 app to Edgio.
 
 ## Demo
 
 https://layer0-docs-layer0-vue3-example-default.layer0-limelight.link
```

Comment:
i expect it to be deployed under old url

Reply:
Yup, I think @tristanlee85 is in process of migrating to other repo

Reply:
ok